src/components/Views/ObservationEditor/ObservationEditor.js

```diff
@@ -439,13 +445,38 @@ class ObservationEditor extends React.Component<
     Keyboard.dismiss();
   };
 
+  handleModalCancel = () => {
+    const {
+      clearSelectedObservation,
+      goToMainCameraView,
+      goToMapView,
+      hideCancelModal,
+      observationSource,
+      updateObservation
+    } = this.props;
+    hideCancelModal();
```

Comment:
let do a space above just to keep logic and const declarations separate

Reply:
👍